sources/TileDB.CSharp/ArraySchema.cs

```diff
@@ -208,8 +233,10 @@ public FilterList CoordsFilterList()
         {
 
             tiledb_filter_list_t* filter_list_p;
-            _ctx.handle_error(Methods.tiledb_array_schema_get_coords_filter_list(_ctx.Handle, _handle, &filter_list_p));
-            return new FilterList(_ctx, filter_list_p);
+            using var ctxHandle = _ctx.Handle.Acquire();
+            using var handle = _handle.Acquire();
+            _ctx.handle_error(Methods.tiledb_array_schema_get_coords_filter_list(ctxHandle, handle, &filter_list_p));
+            return new FilterList(_ctx, FilterListHandle.CreateUnowned(filter_list_p));
```

Comment:
An unowned handle means that if we call `Dispose` it will not be released from the native side. That's what the pre-existing implicit conversion operator did. Is that surely the correct behavior? If not I will fix it in a future PR.

Reply:
It seems to me that this is unintended, but worth checking with @antalakas if there's a reason for this I may be unaware of.